selfdrive/test/process_replay/fuzzy_generation.py

```diff
@@ -0,0 +1,69 @@
+import hypothesis.strategies as st
+import random
```

Comment:
let's move this to selfdrive/test/ and need to add to release/files_common

Reply:
I will make a separate PR for the test_car_interface stuff since it needs an agnos update